palindrome.c

```diff
@@ -0,0 +1,23 @@
+#include <stdio.h>
+#include <string.h>
+
+/*initial left = 0, initial right = strlen(w)-2*/
+int pal(char w[100], int left, int right){
+	if(strlen(w)/2==left || strlen(w)/2==right || right==left)
```

Comment:
For some reason, your code doesn't work on three letter words. It always returns that it's a palindrome when it's clearly not (i.e. "cat"). I ~think~ it's because it doesn't compare the the first and last letter in this case because of the way you access them?